README.md

```diff
@@ -7,100 +9,92 @@
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=peterhuene/azure-functions-rs)](https://dependabot.com)
 [![license](https://img.shields.io/crates/l/azure-functions.svg)](https://github.com/peterhuene/azure-functions-rs/blob/master/LICENSE)
 
-This is an early-stage framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
+An early-stage framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
 in [Rust](https://www.rust-lang.org/).
 
-## Disclaimer
-
-Although the maintainer of this project is a Microsoft employee, this project is not an officially recognized Microsoft product and is not an endorsement of any future product offering from Microsoft.
-
-_Microsoft and Azure are registered trademarks of Microsoft Corporation._
+> :triangular_flag_on_post: **Disclaimer**  
+> Although the maintainer of this project is a Microsoft employee, this project is not an officially recognized Microsoft product and is not an endorsement of any future product offering from Microsoft.  
+> _Microsoft and Azure are registered trademarks of Microsoft Corporation._
 
-## Example
```

Comment:
Just as a personal preference, I like seeing a very small, hello world type example to get a sense of what things look like.  Much how Rocket uses a hello world example on the website and their README right at the top.

Would you mind putting the hello world example back above the `Get Started` section?  Thanks!

Reply:
Sounds like a good idea! I’ll put it back